websites/visualization/src/app/providers.tsx

```diff
@@ -1,47 +1,25 @@
 'use client'
 
 import { CacheStore, CacheStoreProvider } from '@suspensive/cache'
-import { DevMode, Suspensive, SuspensiveProvider } from '@suspensive/react'
+import { DefaultProps, DefaultPropsProvider } from '@suspensive/react'
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools'
 import { type PropsWithChildren, useState } from 'react'
 import { Spinner } from '~/components/uis'
 
 export const Providers = ({ children }: PropsWithChildren) => {
-  const cacheStore = useState(new CacheStore())[0]
-  const queryClient = useState(
-    () =>
-      new QueryClient({
-        defaultOptions: {
-          queries: {
-            retry: 0,
-          },
-        },
-      })
-  )[0]
-  const suspensive = useState(
-    () =>
-      new Suspensive({
-        defaultProps: {
-          delay: {
-            ms: 1200,
-          },
-          suspense: {
-            fallback: <Spinner />,
-          },
-        },
-      })
-  )[0]
+  const cacheStore = useState(() => new CacheStore())[0]
+  const queryClient = useState(() => new QueryClient({ defaultOptions: { queries: { retry: 0 } } }))[0]
+  const defaultProps = useState(() => new DefaultProps({ delay: { ms: 1200 }, suspense: { fallback: <Spinner /> } }))[0]
```

Comment:
Do we need to use uppercase?

```suggestion
  const defaultProps = useState(() => new DefaultProps({ Delay: { ms: 1200 }, Suspense: { fallback: <Spinner /> } }))[0]
```

Reply:
I think they should be in lowercase. Users are generally more accustomed to seeing uppercase when it refers to a class or component. Considering this, since both "suspense" and "delay" are not directly classes or components, they should be in lowercase to be more familiar and approachable for users.

Reply:
But as you know, why I raise this naming issue is that we use uppercases for naming of each components(Suspense, Delay). If we are using uppercase for component naming, Isn't it proper that we use uppercase for target component field naming of defaultProps too?

Isn't it good? I thought it as consistency problem

Reply:
What I mean is that I don't think there is a reason to use lowercase letters, rather than that it has to be capitalized. I think it would be better to use the same capitalization for the names of components, from an intuitive perspective.

I'm having a hard time deciding.

Reply:
Ah, I understand now. You meant that ms and fallback should be specified as default values for each Suspense and Delay component, so they should be in uppercase. If that’s the intention, I think using uppercase is more intuitive. I believe it’s a good idea!

Reply:
@gwansikk @kangju2000 @bigsaigon333 I’m curious about what others think as well. What are your thoughts?

Reply:
I think using lowercase is better. This is because it's not rendering an actual component but passing an option. Lowercase is commonly used for options, and using uppercase might make users hesitate and think they need to provide a value in the form of a component. (Of course, the type is specified, but I think it's better to make it clear that it's an option.)

Reply:
Before that, I'm curious why you want to change it to a more intuitive name. `DefaultProps` seems like a generic component without context. Could you explain in detail why you made this change? @manudeli

Reply:
In my opinion, since the interface is named DefaultProps, I think users will naturally understand it as specifying the default values for a particular component. If it is perceived as I think, then using capitalized names, such as Delay or Suspense, to specify the options would align better with the naming of the interface.

Reply:
@kangju2000, @gwansikk 

- https://github.com/toss/suspensive/pull/1160#discussion_r1695109541
- https://github.com/toss/suspensive/pull/1160#discussion_r1695116805

I want to check whether you guys read these comments. I don't think there is a absolute right choice, but I think we should check each other's thoughts and decide whether to capitalize or lowercase. Let me know whether you understood intention when using uppercase.

## Wrap up

### Uppercase
```tsx
const defaultProps = new DefaultProps({
  Suspense: { // Uppercase: Same with target component
    fallback: <Spinner />
  },
  ...
})

const Example = () => {
  return (
    <DefaultPropsProvider defaultProps={defaultProps}>
      <Suspense /> // default fallback should be <Spinner />
    </DefaultPropsProvider>
  )
}
```

### Lowercase
```tsx
const defaultProps = new DefaultProps({
  suspense: { // Lowercase: Different with target component
    fallback: <Spinner />
  },
  ...
})

const Example = () => {
  return (
    <DefaultPropsProvider defaultProps={defaultProps}>
      <Suspense /> // default fallback should be <Spinner />
    </DefaultPropsProvider>
  )
}
```

Reply:
Oh I agree, I realized it after seeing your example, **uppercase** would be more consistent and natural in that case.

Injected as `DefaultProps.TargetComponent.props(options)` is more natural

Reply:
After @kangju2000 approving renaming, I'll capitalize as below

```tsx
const defaultProps = new DefaultProps({
  Suspense: { // Uppercase: Same with target component
    fallback: <Spinner />
  },
  ...
})

const Example = () => {
  return (
    <DefaultPropsProvider defaultProps={defaultProps}>
      <Suspense /> // default fallback should be <Spinner />
    </DefaultPropsProvider>
  )
}

```

Reply:
@manudeli 
I agree as well. It seems better to keep the keys in uppercase to indicate that they represent components.